blocks/header/header.css

```diff
@@ -91,7 +92,7 @@ header {
   position: relative;
   top: 10px;
   margin-left: 25px;
-  margin-top: -9px !important; 
+  margin-top: -5px !important; 
```

Comment:
@johan-t What is the reason for using `!important`?

Reply:
@vtsaplin good question, I have removed it since it serves no purpose